.eslintrc.cjs

```diff
@@ -135,6 +154,14 @@ module.exports = {
         ],
       },
     },
+    {
+      // Modules with exports that must be resumable
+      files: ['packages/orchestration/src/exos/**'],
```

Comment:
Picking which rules to apply based on something outside the text of the module seems like action-at-a-distance. I'd prefer a `@resumable-check` marker a la `@ts-check` and `@jessie-check`. But I suppose that's more expensive, and preferences differ.

Reply:
yeah a tag like that is relatively expensive.

I could move the rules into an `eslint-config.cjs` in the package. `eslint-config` package has it like that, but even though it's closer it makes it harder to see what all the rules are in the repo.

Another thought I had was to have a file path convention like `<module-name>.resumable.js` or `src/membrane-exos` but that was too clunky.